packages/picasso/src/Amount/Amount.tsx

```diff
@@ -14,6 +14,10 @@ export interface Props extends TypographyProps {
   currency?: string
   /** Locale identifiers are case-insensitive ASCII. However, it's conventional to use title case (first letter capitalized, successive letters lower case) for script code, upper case for region codes, and lower case for everything else. */
   locale?: string
+  /** The minimum number of fraction digits to display **/
+  minimumFractionDigits?: number
+  /** The maximum number of fraction digits to display **/
+  maximumFractionDigits?: number
```

Comment:
```suggestion
  /** The minimum number of fraction digits to display */
  minimumFractionDigits?: number
  /** The maximum number of fraction digits to display */
  maximumFractionDigits?: number
```

Looks like extra asterisks here cause them to be the part of a prop description in storybook
![image](https://user-images.githubusercontent.com/18409292/196157652-85c47973-fb4a-4149-848b-389e899927f0.png)